packages/onboarding/src/utils/flows.ts

```diff
@@ -35,3 +42,19 @@ export const isTailoredSignupFlow = ( flowName: string | null ) => {
 				ECOMMERCE_FLOW === flowName )
 	);
 };
+
+export const isCurrentPlanFlow = ( flowName: string | null ) => {
+	return Boolean(
+		flowName &&
+			[
+				ECOMMERCE_FLOW,
+				BUSINESS_FLOW,
+				PREMIUM_FLOW,
+				PERSONAL_FLOW,
+				ECOMMERCE_MONTHLY_FLOW,
+				BUSINESS_MONTHLY_FLOW,
+				PREMIUM_MONTHLY_FLOW,
+				PERSONAL_MONTHLY_FLOW,
```

Comment:
We also need to include `onboarding` flow otherwise the widget wouldn't show in logged-out `/start`.

Reply:
In case you want to include `/start/free`, then that needs to be included here too.

Reply:
> We also need to include `onboarding` flow otherwise the widget wouldn't show in logged-out `/start`.

Thanks, added in the `Signup.shouldShowZendeskPresalesChat()`